draft-ietf-tls-tls13.md

```diff
@@ -1720,43 +1704,34 @@ Here:
 -  "extension_data" contains information specific to the particular
   extension type.
 
-The initial set of extensions is defined in {{RFC6066}}.
 The list of extension types is maintained by IANA as described in
 {{iana-considerations}}.
 
-An extension type MUST NOT appear in the ServerHello or HelloRetryRequest
-unless the same extension type appeared in the corresponding ClientHello
-(with the exception of "cookie", as noted in {{cookie}}).
-If a client receives an extension type in ServerHello or HelloRetryRequest
-that it did not request in the associated ClientHello, it MUST abort the
-handshake with an "unsupported_extension" fatal alert.
-
-Nonetheless, "server-oriented" extensions may be provided within
-this framework. Such an extension (say, of type x) would require the client to
-first send an extension of type x in a ClientHello with empty extension_data to
-indicate that it supports the extension type. In this case, the client is
-offering the capability to understand the extension type, and the server is
-taking the client up on its offer.
-
-When multiple extensions of different types are present in the ClientHello or
-ServerHello messages, the extensions MAY appear in any order, with the
-exception of "pre_shared_key" {{pre-shared-key-extension}} which MUST
-be last. There MUST NOT be
-more than one extension of the same type.
-
-Finally, note that extensions can be sent both when starting a new session and
-when in resumption-PSK mode. A client that requests session
-resumption does not in general know whether the server will accept this
-request, and therefore it SHOULD send the same extensions as it would send
-normally.
-
-In general, the specification of each extension type needs to describe the
-effect of the extension both during full handshake and session resumption. Most
-current TLS extensions are relevant only when a session is initiated: when an
-older session is resumed, the server does not process these extensions in
-ClientHello, and does not include them in ServerHello. However, some
-extensions may specify different behavior during session resumption.
-[[TODO: update this and the previous paragraph to cover PSK-based resumption.]]
+The client sends its extensions in the ClientHello. The server MAY
+send extensions in the ServerHello, EncryptedExtensions, Certificate,
+and HelloRetryRequest messages. The table in {{iana-considerations}}
+indicates where a given extension may appear. If the client receives
+an extension which is not specified for a given message it MUST
+abort the handshake with an "illegal_parameter" alert.
+
+The server MUST NOT send any extensions which did not appear in the
+corresponding ClientHello (with the exception of "cookie", as
+described in {{cookie}}). If the client received an extension which
+it did not offer in its ClientHello, it MUST abort the handshake with an
+"unsupported_extension" alert. "server-oriented" extensions are supported by
```

Comment:
Missing capitalization for first word of sentence. I suggest just dropping the quotes and the dash.